test_suite/test_validators.py

```diff
@@ -22,7 +23,7 @@ def test_max_choices_single(self):
         item = SelectMultipleField(
             choices=self.choices, max_length=254, max_choices=1)
         self.assertEqual(item.max_choices, 1)
-        self.assertIsInstance(item.validators[1], MaxChoicesValidator)
+        self.assertIsInstance(item.validators[2], MaxChoicesValidator)
```

Comment:
I don't think so. Django will call the validators in the list sequentially. They just need to be there in order to be used by Django validation system.
And it's no issue the position, since validators should not (and I've never seen) be used or expected in a certain position of a list. Except, of course, this particular case of the unit test.